features/multimedia.md

```diff
@@ -0,0 +1,100 @@
+---
+layout: documentation
+title: Multimedia
+---
+
+{% include base.html %}
+
+# Multimedia
+
+## Audio
+
+openHAB is able to play sound either from the file system (files need to be put in the folder `conf/sounds`), from urls (e.g. Internet radio streams) or generated by text-to-speech engines (which are available as optional [Voice add-ons](../addons/voice.html)).
```

Comment:
Should the link (and one more below) point to `{{base}}/addons/voice.html` to keep it in line with eclipse docs and other docs pages? Especially if this may get moved to another place.